sklearn/datasets/tests/test_samples_generator.py

```diff
@@ -435,6 +435,42 @@ def test_make_blobs_error():
         make_blobs(n_samples, centers=3)
 
 
+def test_make_blobs_memory_usage():
+    try:
+        import memory_profiler
+
+        has_memory_profiler = True
+    except:
+        has_memory_profiler = False
+
+    if not has_memory_profiler:
+        pytest.skip("memory_profiler is not available.")
```

Comment:
would this simplification work ?
```py
try:
    import memory_profiler
except ImportError:
    pytest.skip("memory_profiler is not available.")
```

Reply:
`memory_profiler = pytest.importorskip("memory_profile")`